packages/lit-dev-content/site/tutorials/content/working-with-lists/00.md

```diff
@@ -0,0 +1,24 @@
+This tutorial will be exploring different ways of rendering a list of items in
+Lit!
+
+Lit makes it easy to render any lists of items. It has built-it support for
+iterables of any supported values. This, along with Lit's ability to render
+nested templates and helpful built-in directives, allow for rendering any
+variety of lists.
```

Comment:
```suggestion
Lit makes it easy to render lists of items, with has built-it support for
iterables: pass an array (or other iterable) to a child expression and Lit will render each item in the array. This, combined with Lit's ability to render nested templates, and JavaScript patterns for transforming lists of data into lists of templates, allows for very flexible list handling.
```

^ Not sure if this is too complicated or not, but I thought it might summarize list support quickly.